### PR TITLE
Defect#43827 - Suspended execution clean

### PR DESCRIPTION
**Pull request by Razvan-Morar**

### CODE REVIEW

**Comment by Razvan-Morar** [comment]
reopened pr after rebase

File: engine/orchestrator/score-orchestrator-impl/src/main/java/io/cloudslang/orchestrator/repositories/SuspendedExecutionsRepository.java [hunk 1]
```diff
@@ -34,4 +37,16 @@
 
     @Query("from SuspendedExecution se where se.numberOfBranches=size(se.finishedBranches)")
     public List<SuspendedExecution> findFinishedSuspendedExecutions(Pageable pageRequest);
```
Comment (cristian-nitoiu) [inline engine/orchestrator/score-orchestrator-impl/src/main/java/io/cloudslang/orchestrator/repositories/SuspendedExecutionsRepository.java]: remove public

File: package/score-all/src/main/java/io/cloudslang/schema/EngineBeanDefinitionParser.java [hunk 2]
```diff
@@ -103,6 +104,7 @@
 		put(QueueCleanerServiceImpl.class, null);
 		put(QueueStateIdGeneratorServiceImpl.class, null);
         put(ScoreTriggeringImpl.class,null);
```
Comment (cristian-nitoiu) [inline package/score-all/src/main/java/io/cloudslang/schema/EngineBeanDefinitionParser.java]: ocd

File: engine/orchestrator/score-orchestrator-impl/src/main/java/io/cloudslang/orchestrator/services/SuspendedExecutionCleanerServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+    private static final Logger logger = Logger.getLogger(SuspendedExecutionCleanerServiceImpl.class);
+
+    @Override
+    @Transactional
+    public void cleanupSuspendedExecutions() {
+        try {
+            cleanupSuspendedExecutions(MAX_BULK_SIZE);
+        } catch (Exception e) {
+            logger.error("suspended execution cleaner job failed!", e);
```
Comment (George-Giloan) [inline engine/orchestrator/score-orchestrator-impl/src/main/java/io/cloudslang/orchestrator/services/SuspendedExecutionCleanerServiceImpl.java]: `"suspended execution**s**`

File: engine/orchestrator/score-orchestrator-impl/src/main/java/io/cloudslang/orchestrator/repositories/SuspendedExecutionsRepository.java [hunk 4]
```diff
@@ -34,4 +37,16 @@
...
+
+    @Query("select se.executionId from SuspendedExecution se " +
```
Comment (George-Giloan) [inline engine/orchestrator/score-orchestrator-impl/src/main/java/io/cloudslang/orchestrator/repositories/SuspendedExecutionsRepository.java]: This is sooo weird!

**Pull request merged**